src/fsharp/CheckDeclarations.fs

```diff
@@ -30,6 +30,7 @@ open FSharp.Compiler.Syntax.PrettyNaming
 open FSharp.Compiler.SyntaxTreeOps
 open FSharp.Compiler.Text
 open FSharp.Compiler.Text.Range
+open FSharp.Compiler.Xml
```

Comment:
I didn't notice the addition of this namespace.  I don't think we should be using it as it's not about XML as a thing, but about F# documentation

In any case I'd like to make sure all FCS API changes get carefully reviewed - please make sure I sign off on them for now - we need this to be iterating towards being fully stable

Reply:
I can see the logic in moving out of "Syntax" since it's not purely syntax but also additional logic.

Perhaps either of these:

    namespace FSharp.Compiler.XmlDocumentation
    namespace FSharp.Compiler.XmlDocs

though we could also look forward to if we one day support Markdown docs, in which case

    namespace FSharp.Compiler.CodeDocumentation

may be best (to also have corresponding types like `MarkdownDoc` etc.)